docs/100-upgrade/migration-guides/3.3-3.4.mdx

```diff
@@ -173,6 +173,23 @@ export default defineExtension({
 
 :::
 
+### Refactor of `routes`
+
+In this version we updated how `routes` parameter from `defineRemixExtension`
+was resolve. You can now pass directly `import.meta.url` and the routes from
+your `/routes` directory will automatically be resolved:
+
+```js
+{
+  name: "acme",
+  theme: "./extensions/acme/theme",
+  routes: import.meta.url,
+}
+```
+
+For more information, see
+[`defineRemixExtension`'s API reference](/docs/3.x/api-reference/front-commerce-remix/defineRemixExtension#routes)
+
```

Comment:
2 more things must be documented here:
- the fact that extra `root.tsx` in extensions can now be removed
- same for the imports from `UNSTABLE_routes` (and the definitions).

Also, I think that [this section](https://developers.front-commerce.com/docs/3.x/guides/extend-route-layout#re-use-an-existing-layout-route) should be updated too.